src/lib/ReactCore.rei

```diff
@@ -180,7 +179,7 @@ module Make:
     /***
      * Log of operations performed to update an instance tree.
      */
-    module UpdateLog: {type t; let create: unit => t;};
+    module UpdateLog: {type t; type topLevelUpdate; let create: unit => t;};
```

Comment:
I think that `topLevelUpdate` type should be colocated with the `update` function since it's specific to it.